src/Mail/Drivers/Mailgun.js

```diff
@@ -49,7 +49,7 @@ class MailGunTransporter {
    * @return {String}
    */
   get endpoint () {
-    return `https://api.mailgun.net/v3/${this.config.domain}/messages.mime`
+    return `https://api.${this.config.region.length > 0 ? this.config.region.toLowerCase() + '.' : ''}mailgun.net/v3/${this.config.domain}/messages.mime`
```

Comment:
I believe this can be handled in a better way, knowing the fact that `region` can be `undefined` as well.

```js
const urlParts = ['api']
if (typeof (this.config.region) === 'string') {
  urlParts.push(this.config.region.toLowerCase)
}

https://${urlParts.join('.')}mailgun.net/v3/${this.config.domain}/messages.mime
```

Reply:
Hello, it really looks better. Sorry for delay with answer.